state/src/item/text.rs

```diff
@@ -22,7 +22,7 @@ use rlp::{Decodable, DecoderError, Encodable, RlpStream, UntrustedRlp};
 use crate::CacheableItem;
 
 /// Text stored in the DB. Used by Store/Remove Action.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, Serialize, PartialEq)]
```

Comment:
Would you define new type on the RPC module?

Reply:
Fixed!